pkg/build/cmd/rpm.go

```diff
@@ -291,7 +291,7 @@ func signRPMRepo(repoRoot string, cfg PublishConfig) error {
 		PrimaryKey: pubKey,
 		PrivateKey: privKey,
 		Identities: map[string]*openpgp.Identity{
-			uid.Id: &openpgp.Identity{
+			uid.Id: {
```

Comment:
so, we have no idea why this shows up in the diff 😄 . the version "after" is what is currently in the main-branch already.

Reply:
I guess this could be removed if we/ @matyax rebases the branch on main?

Reply:
Probably, but this is a feature branch from an old feature branch with multiple main merges. Rebasing would be catastrophic.